File: facturacion-energia/src/main/java/com/co/facturacion/energia/aplicacion/bean/Solicitud.java
package com.co.facturacion.energia.aplicacion.bean;

import java.io.Serializable;

public class Solicitud implements Serializable{

	private static final long serialVersionUID = 6945613053037830964L;
	
	private SolicitudData solicitudData;
	private InformacionServicio informacionServicio;
	
	public SolicitudData getSolicitudData() {
		return solicitudData;
	}
	
	public InformacionServicio getInformacionServicio() {
		return informacionServicio;
	}
	
	public void setSolicitudData(SolicitudData solicitudData) {
		this.solicitudData = solicitudData;
	}
	
	public void setInformacionServicio(InformacionServicio informacionServicio) {
		this.informacionServicio = informacionServicio;
	}
	
}

File: facturacion-energia-frontend/src/app/modelo/solicitud.ts
import { SolicitudData } from "./solicitudData";
import { InformacionServicio } from "./informacionServicio";

export class Solicitud{

    solicitudData:SolicitudData;
    informacionServicio:InformacionServicio;

    constructor(solicitudData:SolicitudData, informacionServicio:InformacionServicio){

        this.solicitudData = solicitudData;
        this.informacionServicio = informacionServicio;

    }

}